helpers/update-site/documentation.js

```diff
@@ -564,14 +569,25 @@ const updateMarkdownlinks = (content, filePath) => {
             val[0].replace(/(index|readme)\.md/i, '') :
             val[0].replace('.md', '/');
 
+        /*
+         * An github url is found for a md file.
+         * In this case we need to replace it with the a valid
+         * url in the website.
+         */
+        if (val[2].startsWith('https')) {
```

Comment:
It is a bit weird to say the link is for github and then check just for https.

```suggestion
        if (val[2].startsWith('https://github.com/')) {
```